### PR TITLE
Ensure lb has latest source

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

File: AppController/djinn.rb [hunk 1]
```diff
@@ -1729,22 +1719,20 @@ def restart_versions(versions_to_restart)
...
-            " from #{version_key}: #{except_trace}.")
-          # This specific exception may be a JSON parse error.
-          error_msg = "ERROR: Unable to parse app.yaml file for " +
-                      "#{version_key}. Exception of #{except.class} with " +
-                      "message #{except.message}"
-          place_error_app(version_key, error_msg)
-        end
-      }
-    end
+    versions_to_restart.each{ |version_key|
```
Comment (cdonati) [inline AppController/djinn.rb]: Why do machines that aren't load balancers need to parse static data?

Comment (obino) [reply AppController/djinn.rb]: The function is called only by the shadow, so the check is redundant: only shadow (who is a LB) will execute it.

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4246/
faststart build https://ocd.appscale.com:8080/job/Euca-Faststart/1843/console

File: AppController/djinn.rb (hunk 1 above)
Comment (cdonati) [reply AppController/djinn.rb]: Oh, interesting. Does this even need to be run on the shadow?

Comment (obino) [reply AppController/djinn.rb]: yeah only shadow can remove appservers.

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4254/

**Pull request merged**